--- a/font-patcher	2023-12-05 21:07:16.285421643 +0800
+++ b/font-patcher	2023-12-05 21:08:06.346432387 +0800
@@ -16,8 +16,8 @@
 import sys
 import re
 import os
-import argparse
-from argparse import RawTextHelpFormatter
+import configargparse
+from configargparse import RawTextHelpFormatter
 import errno
 import subprocess
 import json
@@ -320,7 +320,7 @@
 
 class font_patcher:
     def __init__(self, args):
-        self.args = args  # class 'argparse.Namespace'
+        self.args = args  # class 'configargparse.Namespace'
         self.sym_font_args = []
         self.config = None  # class 'configparser.ConfigParser'
         self.sourceFont = None  # class 'fontforge.font'
@@ -1844,7 +1844,7 @@
     return False
 
 def setup_arguments():
-    parser = argparse.ArgumentParser(
+    parser = configargparse.ArgumentParser(
         description=(
             'Nerd Fonts Font Patcher: patches a given font with programming and development related glyphs\n\n'
             '* Website: https://www.nerdfonts.com\n'
@@ -1912,7 +1912,7 @@
     # 0       - calculate from font according to OS/2-version-2
     # 500     - set to 500
 
-    # progress bar arguments - https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse
+    # progress bar arguments - https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-configargparse
     progressbars_group_parser.add_argument('--progressbars',         dest='progressbars',     action='store_true',                help='Show percentage completion progress bars per Glyph Set (default)')
     progressbars_group_parser.add_argument('--no-progressbars',      dest='progressbars',     action='store_false',               help='Don\'t show percentage completion progress bars per Glyph Set')
     expert_group.set_defaults(progressbars=True)
