--- a/font-patcher	2024-11-18 02:47:58.000000000 +0800
+++ a/font-patcher	2024-11-18 21:52:08.990116033 +0800
@@ -16,8 +16,8 @@
 import sys
 import re
 import os
-import argparse
-from argparse import RawTextHelpFormatter
+import configargparse
+from configargparse import RawTextHelpFormatter
 import errno
 import subprocess
 import json
@@ -1961,7 +1961,7 @@
 
 def setup_arguments():
     """ Parse the command line parameters and load the config file if needed """
-    parser = argparse.ArgumentParser(
+    parser = configargparse.ArgumentParser(
         description=(
             'Nerd Fonts Font Patcher: patches a given font with programming and development related glyphs\n\n'
             '* Website: https://www.nerdfonts.com\n'
