--- a/font-patcher	2025-04-24 16:22:00.000000000 +0800
+++ a/font-patcher	2025-04-27 11:34:56.404678350 +0800
@@ -16,8 +16,8 @@
 import sys
 import re
 import os
-import argparse
-from argparse import RawTextHelpFormatter
+import configargparse
+from configargparse import RawTextHelpFormatter
 import errno
 import subprocess
 import json
@@ -2017,7 +2017,7 @@
 
 def setup_arguments():
     """ Parse the command line parameters and load the config file if needed """
-    parser = argparse.ArgumentParser(
+    parser = configargparse.ArgumentParser(
         description=(
             'Nerd Fonts Font Patcher: patches a given font with programming and development related glyphs\n\n'
             '* Website: https://www.nerdfonts.com\n'
@@ -2033,13 +2033,13 @@
     parser.add_argument('--careful',                                 dest='careful',          default=False, action='store_true', help='Do not overwrite existing glyphs if detected')
     parser.add_argument('--debug',                                   dest='debugmode',        default=0,     type=int, nargs='?', help='Verbose mode (optional: 1=just to file; 2*=just to terminal; 3=display and file)', const=2, choices=range(0, 3 + 1))
     parser.add_argument('--extension', '-ext',                       dest='extension',        default="",    type=str,            help='Change font file type to create (e.g., ttf, otf)')
-    parser.add_argument('--help', '-h',                              action='help',           default=argparse.SUPPRESS,          help='Show this help message and exit')
+    parser.add_argument('--help', '-h',                              action='help',           default=configargparse.SUPPRESS,    help='Show this help message and exit')
     parser.add_argument('--makegroups',                              dest='makegroups',       default=1,     type=int, nargs='?', help='Use alternative method to name patched fonts (default=1)', const=1, choices=range(-1, 6 + 1))
     parser.add_argument('--mono', '-s',                              dest='forcemono',        default=False, action='count',      help='Create monospaced font, existing and added glyphs are single-width (implies --single-width-glyphs)')
     parser.add_argument('--outputdir', '-out',                       dest='outputdir',        default=".",   type=str,            help='The directory to output the patched font file to')
     parser.add_argument('--quiet', '-q',                             dest='quiet',            default=False, action='store_true', help='Do not generate verbose output')
     parser.add_argument('--single-width-glyphs',                     dest='single',           default=False, action='store_true', help='Whether to generate the glyphs as single-width not double-width (default is double-width) (Nerd Font Mono)')
-    parser.add_argument('--use-single-width-glyphs',                 dest='forcemono',        default=False, action='count',      help=argparse.SUPPRESS)
+    parser.add_argument('--use-single-width-glyphs',                 dest='forcemono',        default=False, action='count',      help=configargparse.SUPPRESS)
     parser.add_argument('--variable-width-glyphs',                   dest='nonmono',          default=False, action='store_true', help='Do not adjust advance width (no "overhang") (Nerd Font Propo)')
     parser.add_argument('--version', '-v',                           action='version',        version=projectName + ': %(prog)s (' + version + ')', help='Show program\'s version number and exit')
     # --makegroup has an additional undocumented numeric specifier. '--makegroup' is in fact '--makegroup 1'.
