index 247c28367..29eeb0fdd 100755
--- a/font-patcher
+++ b/font-patcher
@@ -1448,7 +1448,7 @@ class font_patcher:
                 x_align_distance = self.font_dim['xmin'] - sym_dim['xmin']
                 if sym_attr['align'] == 'c':
                     # Center align
-                    x_align_distance += (self.font_dim['width'] / 2) - (sym_dim['width'] / 2)
+                    x_align_distance += (self.font_dim['width'] * self.get_target_width(stretch) / 2) - (sym_dim['width'] / 2)
                 elif sym_attr['align'] == 'r':
                     # Right align
                     x_align_distance += self.font_dim['width'] * self.get_target_width(stretch) - sym_dim['width']
