--- a/font-patcher	2025-04-24 16:22:00.000000000 +0800
+++ a/font-patcher	2025-04-27 11:40:32.738144574 +0800
@@ -1641,7 +1641,7 @@
                     cell_width = self.font_dim['width']
                 if sym_attr['align'] == 'c':
                     # Center align
-                    x_align_distance += (cell_width / 2) - (sym_dim['width'] / 2)
+                    x_align_distance += (cell_width * self.get_target_width(stretch) / 2) - (sym_dim['width'] / 2)
                 elif sym_attr['align'] == 'r':
                     # Right align
                     # (not really supported with pa scaling and 2x stretch in NFP)
